src/config.yml

```diff
@@ -95,6 +96,10 @@ blocked_commands:
   - 'n:b:/effect:Please use /potion to set effects.'
   - 'n:b:/enderchest:_'
   - 'n:b:/spreadplayers:_'
+  - 'n:b:/border:You are not allowed to edit the world border!.'
+  - 'n:b:/weather:Its always sunny here!'
+  - 'n:b:/eweather:TotalFreedom is sunny not rainy!'
```

Comment:
Again, probably should use the same messages as /weather is the main command and /eweather is just an alias.